shared/src/commonMain/kotlin/org/hyperskill/app/onboarding/data/source/OnboardingCacheDataSource.kt

```diff
@@ -3,9 +3,15 @@ package org.hyperskill.app.onboarding.data.source
 interface OnboardingCacheDataSource {
     fun isOnboardingShown(): Boolean
     fun setOnboardingShown(isShown: Boolean)
+
     fun isParsonsOnboardingShown(): Boolean
     fun setParsonsOnboardingShown(isShown: Boolean)
 
+    fun isFillBlanksInputModeOnboardingShown(): Boolean
+    fun setFillBlanksInputModeOnboardingShown(isShown: Boolean)
+    fun isFillBlanksSelectModeOnboardingShown(): Boolean
+    fun setFillBlanksSelectModeOnboardingShown(isShown: Boolean)
```

Comment:
Maybe here it would be better to use problem type to have only one method for caching. WIth adding new problem types onboardings it would be easier to support